flutter/lib/benchmark/run_mode.dart

```diff
@@ -7,35 +7,40 @@ class BenchmarkRunMode {
   static const _perfLogSuffix = 'performance';
   static const _accuracyLogSuffix = 'accuracy';
 
-  final String mode;
-  final String logSuffix;
+  final String loadgenMode;
+  final String readable;
```

Comment:
What does `readable` mean here? Do you mean `description`?

Reply:
Value of this variable should be human-readable.
Previously it was used only in log file name (which is intended to be handled by vendor developers), now I also use as a log prefix.

Value should be short, to consume less screen space, and it should look nice.
I don't think `description` would fit here, because I would expect description to be more than just one word.

Reply:
Can you add a comment and example value to explain it? I still not understand what is it and for what it's used?

Reply:
I think it would be better to change the name than adding comments with example usage.
But I can't come up with a good name.

If you also don't have any fitting suggestions, I'll add a comment in next PR.

Reply:
Maybe just `logSuffix`?

Reply:
So, just like it was before this PR :-)
I renamed the field because I thought `logSuffix` kinda doesn't fit the fact that it is no longer a true suffix, and the content of the variable will look fitting in any text.

OK, I'll revert it.

Reply:
I only see it used as a log dir suffix in the code, so that name would be good for now.

Reply:
> live logs: `image_classification: performance mode: starting`

Not a log dir suffis

Reply:
I only place I see it used is `logDir = '$logParentDir/${benchmark.id}-${runMode.readable}'`